pkg/cmd/admin/oadm_bashcomp_func.go

```diff
@@ -0,0 +1,16 @@
+package admin
+
+const (
+    bashCompletionFunc = `
```

Comment:
@liggitt you've looked at filecompletion stuff upstream, right?  We're rolling these by hand?